src/Components/Navbar/Navbar.css

```diff
@@ -0,0 +1,79 @@
+.top-bar
+{
+    z-index: 10;
+    border: none;
+    background-color: white;
+    padding: 2rem 4rem;
+    color: var(--background-color);
+    border-bottom: 1px solid var(--background-color);
+}
+
+@media screen and (max-width:1305px){
+    .top-bar
+    {
+        flex-direction: row;
+        gap: 1rem;
+    }
+}
+
+@media screen and (max-width:1015px){
+    .top-bar
+    {
+        flex-direction: column;
+        gap: 1rem;
+    }
+}
+
+#top-bar-brand-name
+{
+    cursor: pointer;
```

Comment:
fix: You can remove # tag as it increases specificity

Reply:
Done. Thanks for pointing it out.